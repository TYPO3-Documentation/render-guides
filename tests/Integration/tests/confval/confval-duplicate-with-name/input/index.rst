=======
Confval
=======

..  confval:: demo
    :type: string
    :default: ``"Hello World"``
    :required: False

    Some Text

See also :confval:`demo` and the config value in the other domain: :confval:`another-demo`.

..  toctree::
    :glob:

    *
