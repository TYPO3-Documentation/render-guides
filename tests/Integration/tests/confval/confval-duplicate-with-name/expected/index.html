<!-- content start -->
                <section class="section" id="confval">
            <h1>Confval<a class="headerlink" href="#confval" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h1>
            <dl class="confval">
    <dt id="confval-demo">
        <code class="sig-name descname"><span class="pre">demo</span></code>
                <a class="headerlink" href="#confval-demo" data-bs-toggle="modal" data-bs-target="#linkReferenceModal" data-id="confval-demo" data-rstCode=":confval:`demo &lt;somemanual:demo&gt;`"  title="Reference this configuration value">¶</a>
        </dt>
    <dd>
                <dl class="field-list simple">
                            <dt class="field-even">Type</dt>
                <dd class="field-even">string
                </dd>
                        <dt class="field-odd">Required</dt>
            <dd class="field-odd"><p>true</p>
            </dd>
                        <dt class="field-odd">Default</dt>
            <dd class="field-odd"><code class="code-inline" translate="no">&quot;Hello World&quot;</code>
            </dd>
            </dl>
                <div class="confval-description">

    <p>Some Text</p>

        </div>
    </dd>
</dl>

    <p>See also <a href="#confval-demo">demo</a> and the config value in the other domain: <a href="anotherDomain.html#confval-another-demo">demo</a>.</p>

            <div class="toctree-wrapper compound">
    <ul class="menu-level">
    <li class="toc-item">
    <a href="anotherDomain.html#confval-in-another-domain">Confval in another domain</a>


</li>
    </ul>
</div>
    </section>
        <!-- content end -->
